data_model/src/query.rs

```diff
@@ -810,6 +812,8 @@ pub mod asset {
     pub struct FindAssetsByAccountId {
         /// [`AccountId`] under which assets should be found.
         pub account_id: EvaluatesTo<AccountId>,
+        /// [`Name`] of key in metadata of [`Asset`]s by which sort.
+        pub sort_by_key: Option<EvaluatesTo<Name>>,
```

Comment:
```suggestion
        pub sort_by_metadata_key: Option<EvaluatesTo<Name>>,
```

Maybe rename to this?